### PR TITLE
[#161411403] Show codes positions for manual payment

### PR DESCRIPTION
**Pull request by matgentili**
Added link for show the modal view. For now the modal view is empty
WIP

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#161411403](https://www.pivotaltracker.com/story/show/161411403): Nello schermo di inserimento manuale dell'avviso di pagamento, Il link "dove si trovano" deve mostrare la spiegazione di dove si trovano i codici nell'avviso cartaceo

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/wallet/payment/CodesPositionManualPaymentModal.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import IconFont from "./../../../components/ui/IconFont";
+
+type Props = {
+  onClose: () => void;
+};
+
+class CodesPositionManualPaymentModal extends React.PureComponent<Props> {
+  public render() {
+    return (
+      <Modal animationType="slide" onRequestClose={this.props.onClose}>
```
Comment (fpersico) [inline ts/screens/wallet/payment/CodesPositionManualPaymentModal.tsx]: Please use the LightModal component.

Comment (matgentili) [reply ts/screens/wallet/payment/CodesPositionManualPaymentModal.tsx]: Okay!

Comment (matgentili) [reply ts/screens/wallet/payment/CodesPositionManualPaymentModal.tsx]: Done.

**Comment by matgentili** [comment]
Hi @matteodesanti , I created this image to help the user understand where are the codes to insert in the manual payment.
What do you think about this design? If you don't like it you can supply one yourself? Thank you!

img

**Comment by matteodesanti** [comment]
ths @matgentili it would be perfect if can zoom into the images. Is that possible?

**Comment by matgentili** [comment]
@matteodesanti yes, I added a library for zoom image ;)

**Pull request merged**